panel/vite.config.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-env node */
 import fs from "fs";
 import path from "path";
+import md5 from "md5";
```

Comment:
I'd recommend [ohash](https://github.com/unjs/ohash) instead.

Why?
- deps-free (opposed to `md5`, which has 3 deps)
- Well tested.
- Used internally by Nuxt 3 (as for all the other UnJS gems).

Reply:
@johannschopplich yea, we kinda need to match what the backend does here. But should be fine since only relevant for development, not in production